README.md

```diff
@@ -361,20 +361,29 @@ This module integrates with Spring Security's OAuth support, all you need is the
 
 ## Proxy
 
-If you're running your application (with this okta-spring-boot dependency) from behind a network proxy, you could add JVM args to your application like:
+If you're running your application (with this okta-spring-boot dependency) from behind a network proxy, you could setup properties for it in application.yml:
+```yaml
+okta:
+  oauth2:
+    proxy:
+      host: "example-proxy.com"
+      port: 7000
+      username: "user"              # optional
+      password: "password"          # optional
+```
+
+or, add JVM args to your application like:
 
 ```bash
--Dhttps.proxyHost=host
--Dhttps.proxyPort=port
--Dhttps.proxyUser="user"          # optional
--Dhttps.proxyPassword="password". # optional 
+-Dokta.oauth2.proxy.host=host
```

Comment:
Please add `username` and `password` properties here.

Reply:
🆗